scripts/templates/component-story.js

```diff
@@ -1,41 +1,54 @@
 module.exports = componentName => [
+  {
+    content: `
+import { Meta, Canvas, Story, ArgsTable } from '@storybook/addon-docs';
+import  ${componentName}  from '../../components/${componentName}';
+
+import { Header } from '../shared';
+
+import { 
+  Default
+} from './${componentName}.stories.jsx';
+
+<Meta title="${componentName}" component={${componentName}}  />
+
+<Header title="${componentName}">
+      Some description for component.
+</Header>
+
+- [Importing and usage](#importing)
+- [API](#api)
+- [Examples](#examples)
```

Comment:
There isn't an examples section further down, what do you think about removing that link?